discovery-provider/src/queries/get_trailing_metrics.py

```diff
@@ -55,5 +60,19 @@ def _get_trailing_app_metrics(session, args):
              .limit(limit)
              .all())
 
+    route_query = route_query.first()
+
     metrics = list(map(lambda m: {"name": m.name, "count": m.count}, query))
+
+    # add unknown count
+    existing_count = reduce(lambda x, y: x + y["count"], metrics, 0)
+    unknown_count = route_query.count - existing_count
+    for i, metric in enumerate(metrics[:]):
```

Comment:
maybe add a comment explaining that you're inserting this in the "right spot" ordered by score. took me a second to read

Reply:
👍